### PR TITLE
Added Posts

### PR DESCRIPTION
**Pull request by MilenaJ-Solicy**

### CODE REVIEW

File: migrations/20230906135621-create-posts.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```
Comment (HakPoghosyan) [inline migrations/20230906135621-create-posts.js]: Why these 2 migrations are written differently

File: migrations/20230906142318-create-user-profile.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
```
Comment (HakPoghosyan) [inline migrations/20230906142318-create-user-profile.js:4]: Why these 2 migrations are written differently

File: migrations/20230906135621-create-posts.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    await queryInterface.createTable('Posts', {
```
Comment (HakPoghosyan) [inline migrations/20230906135621-create-posts.js]: Where is the Post model?

File: models/user.js [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
-  User.init({
-    firstName: DataTypes.STRING,
-    lastName: DataTypes.STRING,
-    email: DataTypes.STRING
-  }, {
-    sequelize,
-    modelName: 'User',
-  });
+
+  User.init(
```
Comment (HakobP-Solicy) [inline models/user.js:11]: Where is the User's migration?

File: models/userprofile.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class UserProfile extends Model {
+    static associate(models) {
+      UserProfile.belongsTo(models.User, { foreignKey: 'userId', as: 'user' });
+    }
+  }
+
+  UserProfile.init(
```
Comment (HakobP-Solicy) [inline models/userprofile.js:11]: Don't need to write a seed for UserProject?

File: migrations/20230906142318-create-user-profile.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    await queryInterface.createTable('UserProfiles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      fullName: {
+        type: Sequelize.STRING
```
Comment (HakPoghosyan) [inline migrations/20230906142318-create-user-profile.js:13]: Need to have tables for all SQL relationships

File: migrations/20230906142318-create-user-profile.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+      },
+      bio: {
+        type: Sequelize.TEXT
+      },
+      userId: {
+        type: Sequelize.INTEGER
+      },
```
Comment (HakPoghosyan) [inline migrations/20230906142318-create-user-profile.js:20]: Need to have a clear Readme file

File: seeders/20230906135817-demo-post.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
```
Comment (HakPoghosyan) [inline seeders/20230906135817-demo-post.js]: Read about seeds

File: enums/status.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+const Status = {
+    PRIVATE: 0,
+    PUBLIC: 1,
+};
+
+module.exports = Status;
```
Comment (HakPoghosyan) [inline enums/status.js]: Add a new line at the end of the file

File: models/post.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    userId: DataTypes.INTEGER,
+    createdAt: DataTypes.DATE,
+    updatedAt: DataTypes.DATE
+  }, {
+    sequelize,
+    modelName: 'Post',
+  });
+  return Post;
+};
```
Comment (HakPoghosyan) [inline models/post.js]: Add a new line at the end of the file

File: migrations/20230907080414-create-post.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
```
Comment (HakPoghosyan) [inline migrations/20230907080414-create-post.js:2]: Add `.vs` and `node_modules` in Git ignore

**Pull request merged**